apps/condo/domains/billing/access/AllResidentBillingVirtualReceiptsService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice billing.AllResidentBillingVirtualReceiptsService --type queries`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canAllResidentBillingVirtualReceipts ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return true
```

Comment:
It would be nice to somehow limit access to data. At least for the user group to which this method applies

Reply:
This is going to work exactly as allResidentBillingReceipt query